versioned_docs/version-6.x.x/core-concepts/model-basics.md

```diff
@@ -425,7 +425,9 @@ Foo.init({
 
   // Creating two objects with the same value will throw an error. The unique property can be either a
   // boolean, or a string. If you provide the same string for multiple columns, they will form a
-  // composite unique key.
+  // composite unique key. Note that this does not work with neither Sequelize.define() nor 
+  // QueryInterface.createTable(). Details here: https://github.com/sequelize/sequelize/issues/14039
```

Comment:
`sequelize.define()` is a wrapper around `Model.init()` so this option does work with define

I don't think it's the role of the model documentation to explain how createTable works. That's a concern for a separate documentation dedicated to queryInterface